tower-balance/src/lib.rs

```diff
@@ -1,10 +1,11 @@
 //! Load balancing middlewares.
 
 #![doc(html_root_url = "https://docs.rs/tower-balance/0.3.0-alpha.1")]
-#![deny(missing_docs)]
-#![deny(rust_2018_idioms)]
+#![warn(missing_docs)]
```

Comment:
minor nit: can we combine this to one line?

Reply:
I personally prefer having them one per line to make it explicit which ones we're using. Scanning a line horizontally is a bit more painful. If you feel strongly I can merge them though.

Reply:
I don't really have a strong preference just seems to be the common pattern in tokio https://github.com/tokio-rs/tokio/blob/master/tokio/src/lib.rs#L2

Either way up to you!